src/map/pc.c

```diff
@@ -5561,6 +5561,13 @@ int pc_setpos(struct map_session_data* sd, unsigned short map_index, int x, int
 		pc->setrestartvalue(sd,1);
 	}
 
+	if (map->list[m].flag.noautoloot) { // Disable autoloot
```

Comment:
Does not need this, as this would be doing unnecessary setting of variables to 0, even if they are 0.
pc_isautolooting should be enough

Reply:
Also, this would mean.
Warping from noautoloot map to other maps where autoloot is allowed would make autoloot list empty.

Reply:
removed this part, auto loot list will be remain upon enter/leave the map with `noautoloot` mapflag.